provers/debug/prover.go

```diff
@@ -0,0 +1,157 @@
+package debug
+
+import (
+	"context"
+	fmt "fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v8/modules/core/exported"
+
+	//mocktypes "github.com/datachainlab/ibc-mock-client/modules/light-clients/xx-mock/types"
+	"github.com/hyperledger-labs/yui-relayer/core"
+	"github.com/hyperledger-labs/yui-relayer/log"
+)
+
+func debugFakeLost(ctx context.Context, chain core.Chain, queryHeight exported.Height) error {
+	if val, ok := os.LookupEnv("DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_PROVER"); ok {
+		s := strings.Split(val, " ")
+		if len(s) != 2 {
+			fmt.Printf("malformed DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_PROVER: Expected format: <chainid> <height threshold>\n")
+			return nil
+		}
+		if s[0] == chain.ChainID() {
+			threshold, err := strconv.Atoi(s[1])
+			if err != nil {
+				fmt.Printf("malformed DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_PROVER: %v\n", err)
+				return nil
+			}
+
+			qh := int64(queryHeight.GetRevisionHeight())
+			latestHeight, err := chain.LatestHeight(ctx)
+			if err != nil {
+				return err
+			}
+
+			lh := int64(latestHeight.GetRevisionHeight())
+			if qh+int64(threshold) < lh {
+				return fmt.Errorf("fake missing trie node: %v + %v < %v", qh, threshold, lh)
+			}
```

Comment:
This line breaking style is a bit confusing.
```suggestion
			qh := int64(queryHeight.GetRevisionHeight())

			latestHeight, err := chain.LatestHeight(ctx)
			if err != nil {
				return err
			}
			lh := int64(latestHeight.GetRevisionHeight())

			if qh+int64(threshold) < lh {
				return fmt.Errorf("fake missing trie node: %v + %v < %v", qh, threshold, lh)
			}
```

Reply:
fixed

Reply:
not fixed?